iothub/device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/mqtt/MqttMessaging.java

```diff
@@ -24,6 +24,16 @@ class MqttMessaging extends Mqtt
     private final String publishTopic;
     private final boolean isEdgeHub;
 
+    private static final String BACKSLASH = "/";
+    private static final String POUND = "#";
```

Comment:
I never cared for how we did this when this MQTT layer was first implemented. Remove these constants and just add them to the constants below as needed